src/Modules/CalcPerform.lua

```diff
@@ -747,11 +747,36 @@ local function doActorCharges(env, actor)
 	modDB.multipliers["SpiritCharge"] = output.SpiritCharges
 end
 
+-- Process slow effects/magnitude
+local function applySlowMagnitude(env)
+	local modDB = env.modDB
+	local enemyDB = env.enemyDB
+	if enemyDB:HasMod("INC", nil, "TemporalChainsActionSpeed", nil) then
+		enemyDB:NewMod("Condition:Slowed", "FLAG", true, 'Skill:TemporalChainsPlayer')
+	end
+
+	if enemyDB:Flag(nil, "Condition:Slowed") then
+		local slowMagnitude = modDB:Sum("INC", nil, "EnemySlowMagnitude")  / 100
+		if slowMagnitude > 0 then
+			enemyDB:ScaleAddMod(modLib.createMod("TemporalChainsActionSpeed", "INC", enemyDB:Sum("INC", nil, "TemporalChainsActionSpeed"), "Slow Magnitude"), slowMagnitude)
+			enemyDB:ScaleAddMod(modLib.createMod("ApexOfMomentSlow", "INC", enemyDB:Sum("INC", nil, "ApexOfMomentSlow"), "Slow Magnitude"), slowMagnitude)
+		end
+	end
```

Comment:
Here I wanted to have a kind of flag for slow skills/debuffs to just take these mods and iterate over them. But I just couldn't figure out how to add this flag